experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -257,3 +257,97 @@ function isomorphism(::Type{PermGroup}, W::WeylGroup; set_properties::Bool=true)
 
   return MapFromFunc(W, G, iso, isoinv)
 end
+
+@doc raw"""
+    map_word(w::WeylGroupElem, genimgs::Vector; init = nothing)
+
+If `init` is `nothing` and `word(w) = [UInt(`$i_1$`), ..., UInt(`$i_n$`)]`, then return the product $R_1 R_2 \cdots R_n$
+with $R_j =$ `genimgs[`$i_j$`]`.
+Otherwise return the product $xR_1 R_2 \cdots R_n$ where $x =$ `init`.
+
+The length of `genimgs` must be equal to the rank of the parent of `w`.
+In particular, it must not be empty.
+If `w` is the trivial element, then `init` is returned if it is different
+from `nothing`, and otherwise `one(genimgs[1])` is returned.
+
+See also: [`map_word(::Union{FPGroupElem, SubFPGroupElem}, ::Vector)`](@ref),
+[`map_word(::Union{PcGroupElem, SubPcGroupElem}, ::Vector)`](@ref).
+Note that `map_word(::WeylGroupElem)` does not provide a `genimgs_inv` parameter
+because the generators of a Weyl group are always self-inverse.
```

Comment:
I think this is a mistake, as it precludes writing generic code.

Instead I would accept it anyway, but you can e.g. ignore it:
```suggestion
Note that `map_word(::WeylGroupElem)` accepts but ignores the `genimgs_inv`
keyword argument because the generators of a Weyl group are always self-inverse.
```
(One could also check if `genimgs == genimgs_inv` if both are given, but I wouldn't bother)

Reply:
Makes sense! Done